tests/run_standalone_tests.sh

```diff
@@ -19,79 +19,100 @@ set -e
 # It can be set through the env variable PL_STANDALONE_TESTS_BATCH_SIZE and defaults to 6 if not set
 test_batch_size="${PL_STANDALONE_TESTS_BATCH_SIZE:-6}"
 source="${PL_STANDALONE_TESTS_SOURCE:-"lightning"}"
+# this is the directory where the tests are located
+test_dir=$1 # parse the first argument
+COLLECTED_TESTS_FILE="collected_tests.txt"
+
+#cd ${test_dir}
```

Comment:
Remove comment or actually `cd` into it?

Reply:
```suggestion
```